brands/shop/doityourself.json

```
@@ -423,6 +427,7 @@
     }
   },
   "shop/doityourself|Sodimac": {
+    "countryCodes": ["cl", "ar", "br", "cl", "co", "mx", "pe", "uy"],
```

```suggestion
    "countryCodes": ["cl", "ar", "br", "co", "mx", "pe", "uy"],
```
Duplicate cl code here.